gremlin-test/src/main/resources/org/apache/tinkerpop/gremlin/test/features/map/LTrim.feature

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+@StepClassMap @StepLTrim
+Feature: Step - lTrim()
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectX__feature___test__nullX_lTrim
+    Given the empty graph
+    And the traversal of
+      """
+      g.inject("  feature", " one test", null).lTrim()
+      """
+    When iterated to list
+    Then the result should be unordered
+      | result |
+      | feature |
+      | one test |
+      | null |
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectX__feature__X_lTrim_concatXleft_trimX
+    Given the empty graph
+    And the traversal of
+      """
+      g.inject("  feature  ").lTrim().concat("left_trim")
+      """
+    When iterated to list
+    Then the result should be unordered
+      | result |
+      | feature  left_trim |
+
+  @GraphComputerVerificationInjectionNotSupported
+  Scenario: g_injectXListXa_bXX_lTrim
+    Given the empty graph
+    And using the parameter xx1 defined as "l[a,b]"
+    And the traversal of
+    """
+    g.inject(xx1).lTrim()
+    """
+    When iterated to list
+    Then the traversal will raise an error with message containing text of "The lTrim() step can only take string as argument"
+
+  Scenario: g_V_valuesXnameX_ltrim_concatXleft_trimX
+    Given the empty graph
+    And the graph initializer of
```

Comment:
why not to use modern graph?

Reply:
I wanted to demonstrate trimming of properties with existing leading and trailing spaces, only the empty graph is reset, all other graphs are shared so we shouldn't modify them

Reply:
ahh, it's modified modern graph
sorry, I missed